covalent/_workflow/transportable_object.py

```diff
@@ -48,18 +52,18 @@ def cat(self) -> bytes:
 
         return string_offset + data_offset + self.header + self.object_string + self.data
 
-    def load(serialized: bytes, header_only: bool, string_only: bool) -> "_TOArchive":
-        string_offset = TOArchiveUtils.string_offset(serialized)
-        header = TOArchiveUtils.parse_header(serialized, string_offset)
+    def load(self, header_only: bool, string_only: bool) -> "_TOArchive":
```

Comment:
I think `load()` would be better as a static or class method because it is used as a constructor. Alternatively, you'll need to change how this function is used in `TransportableObject.deserialize()`

Reply:
Agreed, I think I had changed this since it didn't have a `staticmethod` decorator. I've added that now.